src/components/SignoffSummary/index.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { Fragment } from 'react';
+import { makeStyles } from '@material-ui/styles';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import ListSubheader from '@material-ui/core/ListSubheader';
+import Typography from '@material-ui/core/Typography';
+import { object } from 'prop-types';
+
+const useStyles = makeStyles(theme => ({
+  listSubheader: {
+    lineHeight: 1.5,
+    marginBottom: theme.spacing(1),
+  },
+  listWrapper: {
+    display: 'flex',
+  },
+  requiresSignoffsList: {
+    marginRight: theme.spacing(6),
+  },
+}));
+
+function SignoffSummary(props) {
+  const classes = useStyles();
+  const { requiredSignoffs, signoffs } = props;
+  const listOfSignoffs = Object.entries(signoffs);
+
+  return (
+    <div className={classes.listWrapper}>
+      <List
+        dense
+        subheader={
+          <ListSubheader className={classes.listSubheader}>
+            Requires Signoffs From
+          </ListSubheader>
+        }>
+        {Object.entries(requiredSignoffs).map(([role, count], index) => {
+          const key = `${role}-${index}`;
+
+          return (
+            <ListItem key={key} dense className={classes.requiresSignoffsList}>
+              <ListItemText
+                primary={`${count} member${count > 1 ? 's' : ''} of ${role}`}
+              />
+            </ListItem>
+          );
+        })}
+      </List>
+      {listOfSignoffs && Boolean(listOfSignoffs.length) && (
+        <List
+          dense
+          subheader={
+            <ListSubheader className={classes.listSubheader}>
+              Signed By
+            </ListSubheader>
+          }>
+          <ListItem dense>
+            {listOfSignoffs.map(([username, signoffRole]) => (
+              <ListItemText
+                key={username}
+                primary={
+                  <Fragment>
+                    {username}
+                    {' - '}
+                    <Typography color="textSecondary" variant="caption">
+                      {signoffRole}
+                    </Typography>
+                  </Fragment>
+                }
+              />
+            ))}
+          </ListItem>
+        </List>
+      )}
+    </div>
+  );
+}
+
+SignoffSummary.propTypes = {
+  requiredSignoffs: object.isRequired,
+  signoffs: object.isRequired,
+};
+
+export default SignoffSummary;
```

Comment:
The styling changes suggested above should hopefully do the following change:

#### Before:

![Screen Shot 2019-07-23 at 9 26 36 AM](https://user-images.githubusercontent.com/3766511/61715874-02e15980-ad2c-11e9-8ec6-7b229fd65b77.png)


#### After:

![Screen Shot 2019-07-23 at 9 20 11 AM](https://user-images.githubusercontent.com/3766511/61715834-ecd39900-ad2b-11e9-8c1c-6c03d33a1763.png)